### ISSUE TITLE
Create example scenes highlighting physics

### ISSUE THREAD

**Issue by torchesburn** [webaverse/app#2093]
An example scene with many types of elements that utilise physics

Lets say I want to create a bunch of spheres and have them roll down a ramp. How do we do that?

Another example could be cylinders attached to a joint or arbitrarily to a body, and on the cylinders rolling we would get movement. This to test out drivable vehicle physics.

Plane physics complete with wheels, wings, fins, ailerons, flaps, breaks to reach certain speed and fly, take off and landing included.

Spaceship, with 6dof

F1 Vehicle, reaching crazy speeds

F1ZeroX type vehicle reaching even crazier speeds

TLDR; We need more vehicle physics examples

https://github.com/swift502/Sketchbook

Is it possible to re-use these examples and plug our physics in?

**Comment by yaserOSource** [comment]
https://docs.nvidia.com/gameworks/content/gameworkslibrary/physx/guide/Manual/Vehicles.html#vehicle-sdk-initialization
